add-web-tiled-layer/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,5 @@
+<resources>
+    <string name="app_name">Add Web Tiled Layer</string>
+    <string name="template_uri_stamen">https://stamen-tiles-{subdomain}.a.ssl.fastly.net/terrain/{level}/{col}/{row}.png</string>
+    <string name="stamen_attribution">Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://www.openstreetmap.org/copyright">ODbL</a>.</string>
```

Comment:
These links are not being displayed on the attribution bar. This sample has to link the source of the URI used. Wonder if `attribution` currently has support for `href` links? If not, we can just use plain text here.

Reply:
hmm! this could be an issue..I do not see `attribution` property on WebTiledLayer, instead [this code here](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/72/files#diff-aafed19f5710df3d4497d00980c52b0ec55a206fb8e103d4c0476cb68023ee49R58) sets the `attribution` on the parent class `ImageTiledLayer`.. will check with Changan

Reply:
my bad..the `attribution bar` changes were checked-in 2 days back. https://devtopia.esri.com/runtime/kotlin/pull/1931
using build 3802, you can see the attribution bar